vulnerabilities/tests/test_data/github_api/npm-expected.json

```diff
@@ -0,0 +1,6175 @@
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/webpack/loader-utils/issues/211",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/webpack/loader-utils/blob/d9f4e23cf411d8556f8bac2d3bf05a6e0103b568/lib/interpolateName.js#L38",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/webpack/loader-utils/blob/d9f4e23cf411d8556f8bac2d3bf05a6e0103b568/lib/interpolateName.js#L83",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/webpack/loader-utils/issues/216",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/webpack/loader-utils/commit/36dc86617930a5cf18af51cf3f53d0ee284d2824",
+        "severities": []
+      },
+      {
+        "reference_id": "GHSA-hhq3-ff78-jv3g",
+        "url": "https://github.com/advisories/GHSA-hhq3-ff78-jv3g",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "HIGH",
+            "scoring_elements": ""
+          }
+        ]
+      }
+    ],
+    "date_published": "2022-10-12T12:00:27+00:00"
+  },
+  {
+    "aliases": [
+      "GHSA-98pf-gfh3-x3mp"
+    ],
+    "summary": "Read the Docs vulnerable to Cross-Site Scripting (XSS)",
+    "affected_packages": [
+      {
+        "package": {
+          "type": "npm",
+          "namespace": null,
+          "name": "readthedocs",
+          "version": null,
+          "qualifiers": null,
+          "subpath": null
+        },
+        "affected_version_range": "vers:npm/<8.8.1",
+        "fixed_version": null
+      }
+    ],
+    "references": [
+      {
+        "reference_id": "GHSA-98pf-gfh3-x3mp",
+        "url": "https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-98pf-gfh3-x3mp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "MODERATE",
+            "scoring_elements": ""
+          }
+        ]
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/readthedocs/readthedocs.org/commit/b0ae626acd13882170ec5888e35f3ef2e48e6ff6",
+        "severities": []
+      },
+      {
+        "reference_id": "GHSA-98pf-gfh3-x3mp",
+        "url": "https://github.com/advisories/GHSA-98pf-gfh3-x3mp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "MODERATE",
+            "scoring_elements": ""
+          }
+        ]
+      }
+    ],
+    "date_published": "2022-11-10T16:02:51+00:00"
+  },
+  {
+    "aliases": [
+      "GHSA-3qmc-2r76-4rqp"
+    ],
+    "summary": "Redwood is vulnerable to account takeover via dbAuth \"forgot-password\" ",
+    "affected_packages": [
+      {
+        "package": {
+          "type": "npm",
+          "namespace": null,
```

Comment:
The namespace is `@redwoodjs`

Reply:
There is another issue that https://github.com/redwoodjs/redwood/security/advisories/GHSA-3qmc-2r76-4rqp and (https://github.com/advisories/GHSA-3qmc-2r76-4rqp and https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2022/11/GHSA-3qmc-2r76-4rqp/GHSA-3qmc-2r76-4rqp.json ) are different. Is the graphql data from GH exactly the same or not exactly the same as the JSON?

Reply:
@pombredanne for the namespace issue I have already raised a PR at packageurl-python https://github.com/package-url/packageurl-python/pull/106

Reply:
The GHSA reported here is GHSA-f825-f98c-gj3g and not GHSA-3qmc-2r76-4rqp.

Reply:
We are going to have a data migration issue then. This should be tracked as a high priority issue and will likely need to be an improver or a data migration?
Basically we will need to fix the npm packages that are missing a namespace. But How can we even know about them?

Reply:
@pombredanne bumped the version for package-url .